app/src/main/res/layout/merge_event_details_layout.xml

```diff
@@ -21,6 +21,8 @@
     android:orientation="vertical"
     app:layout_constraintGuide_begin="@dimen/event_details_body_label_margin_start" />
 
+  <!-- region WHEN -->
```

Comment:
These basically let you collapse a whole region, which I found useful while reading this file, I decided to keep them for convenience, can revert in case